lib/mungers/cmd.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const path = require("path")
+const {whichOrUndefined} = require("../which-or-undefined")
+
+function mungeCmd(workingDir, options) {
+  const cmdi = options.args.indexOf('/c')
+  if (cmdi === -1) {
+    return
+  }
+
+  const re = /^\s*("*)([^"]*?\bnode(?:\.exe|\.EXE)?)("*)( .*)?$/
+  const npmre = /^\s*("*)([^"]*?\b(?:npm))("*)( |$)/
+
+  const command = options.args[cmdi + 1]
+  if (command === undefined) {
+    return
+  }
+
+  let m = command.match(re)
+  let replace
+  if (m) {
+    options.originalNode = m[2]
+    // TODO: Remove `replace`: seems unused
```

Comment:
lets drop it, move the `let replace` declaration to the else block and make it const.

Reply:
I agree assuming there's coverage?

Reply:
Leaving this to a follow-up PR.